sdk/python/feast/constants.py

```diff
@@ -116,7 +116,7 @@ class AuthProvider(Enum):
     # Path to certificate(s) to secure connection to Feast Serving
     CONFIG_SERVING_SERVER_SSL_CERT_KEY: "",
     # Default connection timeout to Feast Serving and Feast Core (in seconds)
-    CONFIG_GRPC_CONNECTION_TIMEOUT_DEFAULT_KEY: "3",
+    CONFIG_GRPC_CONNECTION_TIMEOUT_DEFAULT_KEY: "10",
```

Comment:
Had to bump this since some job service methods in turn make calls to AWS/GCP APIs. Sometimes those take a while especially in development env where your job service is not running in the cloud itself.